java/connector-node/README.md

```diff
@@ -30,18 +30,20 @@ This will create a `.tar.gz` file with the Connector Node and all its dependenci
 
 ```
 # unpack the tar file, the file name might vary depending on the version
-cd assembly/target && tar xvf risingwave-connector-1.0.0.tar.gz 
+cd java/connector-node/assembly/target && tar xvf risingwave-connector-1.0.0.tar.gz 
 # launch connector node service
 java -classpath "./libs/*" com.risingwave.connector.ConnectorService
 ```
 
-Sometimes, you need to specify the shared library path. For example, when program want to run class related to java-binding, they need to call shared library file. If not, it will throw exception when running. You need:
+Currently, during the Maven build process, all the required shared libraries are built and added to the JAR file. These shared libraries can then be loaded directly from the Java code.
 
-```
-RISINGWAVE_ROOT=$(git rev-parse --show-toplevel)
-CONNECTOR_ROOT=$JAVA_ROOT/connector-node
-cd ${CONNECTOR_ROOT}/assembly/target && java -classpath "./libs/*" com.risingwave.connector.ConnectorService
-```
+Here are some tips to follow:
+
+If you encounter an error stating that the program cannot access the library, it may be due to merging new features or outdated documentation. In such cases, you will need to manually build the corresponding Java shared library file.
+
+After building the shared library file, move it into the `java/connector-node/assembly/target directory`, make sure to specify the shared library path using the `-Djava.library.path=java/connector-node/assembly/target` flag. This tells Java where to find the required shared library files.
```

Comment:
These paragraph is unnecessary. Anyone successfully build the maven project should be able to run. We don't need explicitly building the shared library any more.

Reply:
removed